src/lib/rest/OrionError.cpp

```diff
@@ -142,3 +142,64 @@ std::string OrionError::render(ConnectionInfo* ciP, const std::string& _indent)
 
   return out;
 }
+
+
+/* ****************************************************************************
+*
+* OrionError::render -
+*
+* FIXME P3: OrionError() render method should be unified (probably the class itself needs
+* a good refactoring, to live in the NGSIv1-NGSIv2 dual world...)
+*
+* Simplified version, without ciP
+*
+*/
+std::string OrionError::render(const std::string& _indent, const std::string apiVersion)
+{
+  //
+  // For API version 2 this is pretty easy ...
+  //
+  if (apiVersion == "v2")
+  {
+    if (details == "Already Exists")
+    {
+      details = "Entity already exists";
```

Comment:
Nah, I'm wrong.
Pretty sure I am thinking of errorStringForV2, which converts `reasonPhrase` for V2, not `details`
[ It's right here a few lines below ... ]

NTC